studio/src/pages/fact-checks/components/FactCheckForm.js

```diff
@@ -339,26 +340,32 @@ function FactCheckForm({ onCreate, data = {}, actions = {}, format }) {
                 </Form.Item>
                 <Form.Item
                   name="excerpt"
-                  label="Excerpt"
                   rules={[{ max: 5000, message: 'Excerpt must be a maximum of 5000 characters.' }]}
                 >
-                  <Input.TextArea rows={4} placeholder="Excerpt" style={{ fontSize: 'medium' }} />
+                  <label>Excerpt</label>
+                  <Input.TextArea rows={4} placeholder="Excerpt"
+                  style={{ width: '100%', marginTop: "0.5rem", fontSize: 'medium' }} />
                 </Form.Item>
-                <Form.Item name="subtitle" label="Subtitle">
-                  <Input placeholder="Subtitle" style={{ fontSize: 'medium' }} />
+                <Form.Item name="subtitle">
+                  <label>Subtitle</label>
+                  <Input placeholder="Subtitle" style={{ width: '100%', marginTop: "0.5rem", fontSize: 'medium' }}  />
                 </Form.Item>
-                <SlugInput />
-                <Form.Item name="published_date" label="Published Date">
-                  <DatePicker />
+                <SlugInput  style={{ width: '100%', marginTop: "0.5rem", fontSize: 'medium' }}/>
+                <Form.Item name="published_date" >
+                  <label>Publish Date</label>
+                  <DatePicker style={{ width: '100%', marginTop: "0.5rem", fontSize: 'medium' }} />
                 </Form.Item>
-                <Form.Item name="categories" label="Categories">
-                  <Selector mode="multiple" action="Categories" createEntity="Category" />
+                <Form.Item name="categories">
+                  <label>Categories</label>
```

Comment:
https://ant.design/components/form . cant we use the layout prop instead of manually adding labels . 

<img width="1235" alt="Screenshot 2023-06-28 at 4 58 34 PM" src="https://github.com/factly/dega/assets/91077192/e568a837-93b5-4269-a31a-cacf0e4119fe">

Reply:
https://github.com/ishpaul207/dega/blame/resdesign-issues/studio/src/pages/fact-checks/components/FactCheckForm.js#L225C7-L242

used already but not working

Reply:
is it an issue on antd or on our end ?

Reply:
Faced this issue quite a few times couldn't figure out reason,